x-hooks/use-bounty.tsx

```diff
@@ -114,13 +115,20 @@ export function useBounty() {
       .then(pullRequests => {
         const extendedPrs = pullRequests.slice(1) as pullRequest[];
         dispatch(changeCurrentBountyData({ ...pullRequests[0], pullRequests: extendedPrs }));
-      });
-
+      })
+      .catch(error => {
+        console.debug("Failed to get database data", error);
+        replace(getURLWithNetwork("/"));
```

Comment:
I understand, but maybe we should redirect the user to the 404 page as it would help the user (and us) better understand what happened [and why we got redirected]

but we can discuss this at the daily meeting because I can be convinced otherwise x)